doc/usage.rst

```diff
@@ -83,6 +83,44 @@ Setting up your target to use a header-only version of ``fmt`` is equally easy::
 
    target_link_libraries(<your-target> PRIVATE fmt::fmt-header-only)
 
+Usage with build2
+=================
+
+
+You can use `build2 <https://build2.org>`_, a dependency manager and a build-system combined, to use ``fmt``:
+
+Currently this package is available in these package repositories:
+
+- **https://cppget.org/fmt/** for released and published versions.
+- `The git repository with the sources of the build2 package of fmt <https://github.com/build2-packaging/fmt.git>`_  for unreleased or custom revisions of ``fmt``.
+
+**Usage:**
+
+- ``build2`` package name: ``fmt``
+- Library target name : ``lib{fmt}``
+
+For example, to make your ``build2`` project depend on ``fmt``:
+
+- Add one of the repositories to your configurations, or in your ``repositories.manifest``, if not already there. For example::
+
+    :
```

Comment:
Is : needed here?

Reply:
Yes this is part of the syntax. See this file as an example: https://github.com/Klaim/wyvern/blob/master/repositories.manifest
(or any build2 project, for example build2 itself: https://github.com/build2/build2/blob/master/repositories.manifest )